.buildkite/hooks/pre-command

```diff
@@ -84,10 +86,28 @@ ENABLED_BEATS_PIPELINES_SLUGS=(
 for slug in "${ENABLED_BEATS_PIPELINES_SLUGS[@]}"; do
   if [[ "$BUILDKITE_PIPELINE_SLUG" == "$slug" ]]; then
     source .buildkite/scripts/setenv.sh
+    if [ "${arch_type}" == "x86_64" ]; then
+      case "${platform_type}" in
+        Linux|Darwin)
+          echo "TBD"
+          ;;
+        MINGW*)
+          echo "-- Fixing CRLF in git checkout --"
```

Comment:
@dliappis I wonder if we could set this in our Windows images, so we don't need to do anything in the repo scripts.

Happy to see if I can make it work for windows.

Reply:
@alexsapran yup this would be an easy one to fix. I suppose we should add it at the global level?

Reply:
> @alexsapran yup this would be an easy one to fix. I suppose we should add it at the global level?

In my opinion - yes

Reply:
I'll remove that when it's migrated into our images

Reply:
I have raised a PR https://github.com/elastic/ci-agent-images/pull/589
Hopefully, soon it will be merged.

Reply:
I have rebuild and pushed a new image, using the latest image should be enough and this is not required anymore

Reply:
> I have rebuild and pushed a new image, using the latest image should be enough and this is not required anymore

@alexsapran it doesn't work as expected.
There are examples issues:
https://buildkite.com/elastic/beats-metricbeat/builds/4088
https://buildkite.com/elastic/beats-xpack-filebeat/builds/709
https://buildkite.com/elastic/beats-xpack-metricbeat/builds/1607
There are result of using commands in the pre-command:
https://buildkite.com/elastic/beats-metricbeat/builds/4078
https://buildkite.com/elastic/beats-xpack-filebeat/builds/699
https://buildkite.com/elastic/beats-xpack-metricbeat/builds/1597

Reply:
Thank @sharbuz I found the problem, which was the fact the BK was running with a specific user. It should be ok now.

Reply:
retrying to a previous failing build now works 
https://buildkite.com/elastic/beats-metricbeat/builds/4088#_